pkg/proto/datadog/remoteconfig/remoteconfig.proto

```diff
@@ -49,6 +49,7 @@ message LatestConfigsRequest {
   repeated string products = 4;
   repeated string new_products = 5;
   repeated Client active_clients = 6;
+  bytes client_state = 10;
```

Comment:
Why 10, as opposed to just ++'ing to 7?

Reply:
Fields are not in order unfortunately

Reply:
Ahhh, minimal diff surrounding context fail on my part. 😁 Thanks!